solutions/instances/outputs.tf

```diff
@@ -14,22 +14,22 @@ output "resource_group_id" {
 
 output "scc_id" {
   description = "SCC instance ID"
-  value       = module.scc.id
+  value       = var.existing_scc_instance_crn == null ? module.scc[0].id : null
 }
 
 output "scc_guid" {
   description = "SCC instance guid"
-  value       = module.scc.guid
+  value       = var.existing_scc_instance_crn == null ? module.scc[0].guid : null
 }
 
 output "scc_crn" {
   description = "SCC instance CRN"
-  value       = module.scc.crn
+  value       = var.existing_scc_instance_crn == null ? module.scc[0].crn : null
```

Comment:
use `existing_scc_instance_crn` instead of `null` here so it will output for existing SCC use case

Reply:
addressed